### PR TITLE
zavihan rokav

### LINKED ISSUE #1529: Update social media cards
- Collections - https://beta.kodadot.xyz/rmrk/collections
image

- Spotlight - https://beta.kodadot.xyz/spotlight
image

- Gallery feed - https://beta.kodadot.xyz/rmrk/gallery
image

- on all Create dialogs - https://beta.kodadot.xyz/rmrk/create - https://beta.kodadot.xyz/rmrk/mint
image

- Series Insights - https://beta.kodadot.xyz/series-insight
image

- Default elsewhere, like at / &c
image
 
### Ref
#1612

### PR DESCRIPTION
**Pull request by yangwao**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [X] Most advanced issue

### Before submitting Pull Request, please make sure:
- [-] My contribution builds **clean without any errors or warnings**
- [-] I've merged recent default branch -- **main-nuxt** and I've no conflicts
- [-] I've tried respect high code quality standards
- [-] I've didn't break any original functionality
- [-] I've posted screenshot of demonstrated change in this PR

### Optional
- [x] I found edge cases -- cosmic rays

### What's new?
- [x] PR closes #1529

### CODE REVIEW

**Review by vikiival — APPROVED** [review]
I believe in your Slovenian accent LGTM 🇸🇮

File: components/landing/Landing.vue [hunk 1]
```diff
@@ -145,7 +145,7 @@ const components = {}
         },
         {
           property: 'og:image',
-          content: 'https://nft.kodadot.xyz/kodadot_carbonless.jpg',
+          content: `${require(`/public/kodadot_card_root.png`)}`,
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply components/landing/Landing.vue:148]: oh no

**Comment by yangwao** [comment]
Apparently. when I add BASE_URL to my local `.env` it works.. but somehow on Netlify not, even I see it's there


image


ohmmmgg just before smashing button i hit very last refresh and it's there

image

**Comment by yangwao** [comment]
image

**Pull request merged**